news/7740.feature

```diff
@@ -0,0 +1 @@
+Fixed ``pyproject.toml`` in ordert o make ``python3 -m pep517.build` work.
```

Comment:
Typo:
```suggestion
Fixed ``pyproject.toml`` in ordert o make ``python3 -m pep517.build`` work.
```

Reply:
Further typo: `ordert o` -> `order to`.

Also, it's not accurate to say that `pep517.build` "doesn't work". Precisely because pip doesn't specify a `build-backend`, we're not using a PEP 517 style source tree layout. So it's not obvious at all that a PEP 517 build tool like `pep517.build` should be expected to be able to build pip.

A claim like "Use PEP 517 layout by specifying `build-backend`, so that pip can be built with tools that don't support the legacy layout" would be more accurate.

Reply:
Thanks, fixed.